target/linux/ath79/patches-5.4/940-ath79-add-qcn550x.patch

```diff
@@ -0,0 +1,35 @@
+--- a/arch/mips/ath79/early_printk.c
++++ b/arch/mips/ath79/early_printk.c
+@@ -121,6 +121,7 @@ static void prom_putchar_init(void)
+ 	case REV_ID_MAJOR_QCA9558:
+ 	case REV_ID_MAJOR_TP9343:
+ 	case REV_ID_MAJOR_QCA956X:
++	case REV_ID_MAJOR_QCN550X:
```

Comment:
Consider separating the board patch into a commit of its own.

Reply:
Done